src/components/loading.jsx

```diff
@@ -5,7 +5,7 @@ export default class Loading extends Component {
   render () {
     return (
       <div className="loading">
-        <object type="image/svg+xml" data={infinityLoaderPath} />
+        <object type="image/svg+xml" data={infinityLoaderPath}>Loading…</object>
```

Comment:
This both makes testing easier in RTL and gives us an accessible fallback to the spinner icon.